distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -234,19 +234,34 @@ chart:provider=default
 #####                   I/O component configurations                              #####
 #######################################################################################
 
-########################## Google Calendar configuration ##############################
+####################### Google Calendar (GCal) configuration ##########################
 #
-# the username and password for Google Calendar Account
-#gcal:username=
-#gcal:password=
+# Before using GCal, you need to have a Google API
+# Console project. The Wiki describes in detail the steps necessary to set 
+# up your Google API Console project, as well as how to obtain the credentials 
+# necessary to complete the information in this file. Once the project is created, 
+# and you've completed the steps described in the Wiki, you need to copy 
+# the "Client ID" and "Client secret" from the Credentials page on 
+# console.developers.google.com
+#
+# Copied from the "Client ID" field on the Credentials page (required)
+#gcal:client_id=
+
+# Copied from the "Client secret" field on the Credentials page (required)
+#gcal:client_secret=
 
-# the url of the calendar feed
-#gcal:url=
+# This is the name you gave to your Google Calendar.  GCal will download calendar 
+# events from this calendar (required)
+#gcal:calendar_name=
```

Comment:
> both gcal and gcal-persistence require a calendar name in the cfg. If not supplied, it doesn't default to anything. It does log errors in the log file, however.

The code does log the error and throw the exception, but the `calendar_name` is initialized to `"primary"` in the class so it didn't make sense -- either have no default for the member, or allow it to be optional.  In any case, I'm fine with requiring the user to explicitly set the calendar, for the reasons you gave.  Yes, please update the .cfg files to make it clear, and thanks!

> Should I include an updated MANIFEST.MF in this PR?

Beat you to it!  #4899 😄